### DISCUSSION TITLE
Unable to Copy as TXT (advanced copy)

### DISCUSSION THREAD

**Discussion by mvwood** [dbeaver/dbeaver#19602] [Q&A]
Unable to copy as TXT rows in the db results with the error shown in screenshot:
![Screenshot 2023-04-04 at 4 07 10 PM](https://user-images.githubusercontent.com/107649630/229908544-bcd8aead-daea-4181-9ece-3c1892d3a165.png)

It's only with one of my databases and the issue started today after updating to version 23.0.1.202303250535. I don't recall my previous version. I uninstalled/reinstalled. 
The issue doesn't happen if I copy as any other format. 

I played around with the copy settings and increased fields significantly. My current settings are as follows. 
![Screenshot 2023-04-04 at 4 16 49 PM](https://user-images.githubusercontent.com/107649630/229910553-aeb0eeb6-8b70-4e0b-b70d-e4dc26819b01.png)
![Screenshot 2023-04-04 at 4 12 08 PM](https://user-images.githubusercontent.com/107649630/229910581-9b54bf14-70d8-469b-a0a6-df4e3d8cb24a.png)

I wasn't able to find this issue for others when searching. I feel like it's just a setting that I need to update, but Even setting batch size to a high count and enabling show nulls didn't resolve. I made sure that use sql fetch size is not selected (wasn't by default). I did enable it and tried as well.

**Comment by LonwoLonwo** [comment]
Could you please provide the log files? You can find instructions here: https://dbeaver.com/docs/wiki/Log-files/. Thank you.

**Comment by LonwoLonwo** [comment]
I think steps to reproduce or video also will be helpful.
